### PR TITLE
Updated the version numbers to 6.9 in docinfo.xml and attributes.adoc…

### PR DESCRIPTION
**Pull request by suyogsainkar**
… for 6.9 GA release

Update attributes.adoc and docinfo files for 6.9
https://bugzilla.redhat.com/show_bug.cgi?id=1951584

Cherry-pick into:

* [x] Foreman 2.3 (Satellite 6.9)
* [ ] Foreman 2.1 (Satellite 6.8)

### CODE REVIEW

**Comment by suyogsainkar** [comment]
@melcorr @lzap could you review and merge these changes (and cherry pick to 2.3) as soon as possible please? Need these updates to make docs ready in pantheon/docs splash page for the GA release. Thanks!

File: guides/doc-Administering_Red_Hat_Satellite/build/en-US/Revision_History.xml [hunk 1]
```diff
@@ -0,0 +1,18 @@
+<?xml version='1.0' encoding='UTF-8'?>
```
Comment (melcorr) [inline guides/doc-Administering_Red_Hat_Satellite/build/en-US/Revision_History.xml:1]: What is this?

File: guides/doc-Administering_Red_Hat_Satellite/build/en-US/master.xml [hunk 2]
```diff
@@ -0,0 +1,6377 @@
...
+<!ENTITY % DOCBOOK_ENTS PUBLIC "-//OASIS//ENTITIES DocBook Character Entities V4.5//EN" "http://www.oasis-open.org/docbook/xml/4.5/dbcentx.mod">
+%DOCBOOK_ENTS;
+]>
+<?asciidoc-toc?><?asciidoc-numbered?><book xmlns="http://docbook.org/ns/docbook" xmlns:xlink="http://www.w3.org/1999/xlink" version="5.0">
+<info>
+<title>Administering {ProjectName}</title><subtitle>A guide to administering Red Hat Satellite.</subtitle>
+
+<date>published {date_my}</date>
+<productname>Red Hat Satellite</productname>
+<productnumber>6.9 Beta</productnumber>
```
Comment (melcorr) [inline guides/doc-Administering_Red_Hat_Satellite/build/en-US/master.xml:14]: Should this be 6.9?

**Review by melcorr — CHANGES_REQUESTED** [review]
This doesn't look right @suyogsainkar

**Comment by suyogsainkar** [comment]
@melcorr sorry, looks like those files got committed due to error. Please ignore. Will open a new PR.

**Pull request closed without merging**